src/Framework/TestCase.php

```diff
@@ -1002,6 +1017,49 @@ public function setPreserveGlobalState(bool $preserveGlobalState): void
         $this->preserveGlobalState = $preserveGlobalState;
     }
 
+    /**
+     * @param string $order
+     */
+    public function setTestRunningOrder($order): void
```

Comment:
Please use scalar type declarations when possible (like here). Also please do not use `@param` annotations etc. when they provide no additional information.

Reply:
Fixed